### ISSUE TITLE
Doesn't work with Vue CLI 3

### ISSUE THREAD

**Issue by genu** [vue-styleguidist/vue-styleguidist#181]
I tried setting up `vue-styleguidist` on an existing Vue CLI 3 project, and it errors out with:

```
Loading webpack config from:
/Users/user/github/m-client/node_modules/@vue/cli-service/webpack.config.js

(node:95255) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
You can now view your style guide in the browser:

  Local:            http://localhost:6060/
  On your network:  http://192.168.1.65:6060/

./src/components/AppButton.vue 1:0
Module parse failed: Unexpected token (1:0)
You may need an appropriate loader to handle this file type.
> <template>
|   <button :style="styles" @click="handleClick">
|     <!-- @slot Use this slot to place the button content -->

Learn how to add webpack loaders to your style guide:
https://github.com/vue-styleguidist/vue-styleguidist/blob/master/docs/Webpack.md

 ERROR  vue-styleguidist exited with code 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I then tried to run the vue cli 3 example from this repo, and the error is:

```
Building style guide...
Error: Cannot find module 'vue-loader'
Error: Cannot find module 'vue-loader'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/user/Desktop/temp/vue-styleguidist/examples/vuecli3/node_modules/vue-styleguidist/scripts/utils/mergeWebpackConfigVueCLI.js:1:81)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/user/Desktop/temp/vue-styleguidist/examples/vuecli3/node_modules/vue-styleguidist/scripts/make-webpack-config.js:14:34)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
 ERROR  vue-styleguidist exited with code 1.
error Command failed with exit code 1.
```

Any idea what I am doing wrong?

**Comment by PhmNgocNghia** [comment]
+1

**Comment by rafaesc** [comment]
@genu @PhmNgocNghia It fixed on version `1.8.3`
